proto/micro_rpc/BUILD

```diff
@@ -33,16 +31,10 @@ java_proto_library(
     deps = [":messages_proto"],
 )
 
-java_lite_proto_library(
```

Comment:
Is there no case in which a java lite proto target may fail to compile, even if other (non-lite) proto target succeed? In other words, do we not lose any checks this way?

Reply:
I don't know if that's possible... java lite libs don't depend on other java lite libs, only on directly on protos. I don't know if there's any proto feature that won't compile as lite. In any case, at the moment all of our java_lite_proto_library rules fail to compile while their non-lite couterparts do compile so we're not detecting that kind of case. With cl/614980287 we'll start to detect them on import.